types/marked-terminal/tsconfig.json

```diff
@@ -13,9 +13,7 @@
         "typeRoots": [
             "../"
         ],
-        "paths": {
-            "marked": [ "marked/v3" ]
-        },
+        "target": "es6",
```

Comment:
```suggestion
```

Reply:
This is ok according to the [docs](https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson):

> You may edit the tsconfig.json to add new test files, to add "target": "es6"